packages/docs/src/styles/components/syntaxHighlighting.scss

```diff
@@ -7,6 +7,7 @@ code {
 pre {
   display: block;
   padding: $spacer-2;
+  overflow-x: scroll;
```

Comment:
Maybe this should be `auto`? Haven't run this yet, but I think setting it to `scroll` will result in there always being a scrollbar, whereas `auto` will only show a scrollbar if the container needs it.

Reply:
Good call :+1: